fe/fe-core/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -510,6 +510,16 @@ public static void tabletReport(long backendId, Map<Long, TTablet> backendTablet
         LOG.info("finished to handle tablet report from backend[{}] cost: {} ms", backendId, (end - start));
     }
 
+    private static void updateControlPublishVersion(long backendId, Map<TTaskType, Set<Long>> runningTasks) {
+        if (!runningTasks.containsKey(TTaskType.PUBLISH_VERSION)) {
+            return;
+        }
+        LOG.debug("current beId {} - {} - {}",
+                backendId, runningTasks, runningTasks.get(TTaskType.PUBLISH_VERSION).size());
```

Comment:
even log is debug,  runningTasks toString will still execute, and it's time cost

Reply:
Log the publish version task count at the end of function 'taskReport'